apps/web/test/api-comments.test.ts

```
@@ -0,0 +1,390 @@
+	auth: {
+		getUser: mock(async () => ({
+			data: { user: { id: '123e4567-e89b-12d3-a456-426614174000' } },
+			error: null,
+		})),
+	},
+}
+
+describe('/api/comments', () => {
+	beforeEach(() => {
+		// Reset all mocks
+		Object.values(mockSupabase).forEach((fn: any) => {
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)